### ISSUE TITLE
Figma import

### ISSUE THREAD

**Issue by katoquro** [refactoringui/heroicons#8]
Hello! Can some one help me with figma import? 
I've tried import in Figma Desktop for Mac and after it they look wrong :(
I have previously used other icon sets and everything was ok

![image](https://user-images.githubusercontent.com/1694937/75423633-2b7ba280-5950-11ea-8578-30a117a7524d.png)

**Comment by codex-zaydek** [comment]
Hey @katoquro, I got curious about this and took a closer.

Here’s the source code for `md-qrcode.svg`:

```svg
<svg fill="none" viewBox="0 0 24 24" stroke="currentColor" class="w-8 h-8">
  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M12 4v1m6 11h2m-6 0h-2v4m0-11v3m0 0h.01M12 12h4.01M16 20h4M4 12h4m12 0h.01M5 8h2a1 1 0 001-1V5a1 1 0 00-1-1H5a1 1 0 00-1 1v2a1 1 0 001 1zm12 0h2a1 1 0 001-1V5a1 1 0 00-1-1h-2a1 1 0 00-1 1v2a1 1 0 001 1zM5 20h2a1 1 0 001-1v-2a1 1 0 00-1-1H5a1 1 0 00-1 1v2a1 1 0 001 1z"></path>
</svg>
```

My best guess is that upon export, the icons were _flattened_ which is confusing Figma. I think Figma expects each of the QR code’s lines to be independent, but here you can see it’s represented as a single path. If I’m not mistaken, _all_ Heroicons were exported like this. It was probably an oversight, nothing that can’t be fixed in the near future.

Contrast this with [Feather icons](https://feathericons.com).

Here’s the source code for `cloud-snow.svg`:

```svg
<svg viewBox="0 0 24 24" width="24" height="24" stroke="currentColor" stroke-width="2" fill="none" stroke-linecap="round" stroke-linejoin="round" class="css-i6dzq1">
	<path d="M20 17.58A5 5 0 0 0 18 8h-1.26A8 8 0 1 0 4 16.25"></path>
	<line x1="8" y1="16" x2="8.01" y2="16"></line>
	<line x1="8" y1="20" x2="8.01" y2="20"></line>
	<line x1="12" y1="18" x2="12.01" y2="18"></line>
	<line x1="12" y1="22" x2="12.01" y2="22"></line>
	<line x1="16" y1="16" x2="16.01" y2="16"></line>
	<line x1="16" y1="20" x2="16.01" y2="20"></line>
</svg>
```

The cloud and snow dots were exported as separate paths or lines. Figma works fine with this icon for reference.

I’ve seen Adam working with Heroicons in Figma before, so there’s probably a pure reference implementation around somewhere. We just don’t have access to it _yet_.

**Comment by codex-zaydek** [comment]
So two things:

It looks like it’s not that hard to correct the icons in Figma. At worst, Figma is connecting dots (e.g. creating new lines), but it’s not distorting the icons more than that from what I can tell.

When I work on my viewer next, I’ll see if I can create a cleaned up Figma reference file.

**Comment by katoquro** [comment]
@codex-zaydek thank you for investigation!

I also thought that there is something during the export. 
Any way having a figma reference sounds great! It could help to see updated and new icons

**Comment by codex-zaydek** [comment]
If you just need helping _seeing_ the icons, you can use https://heroicons.dev. Will follow-up on the reference Figma file soon.

**Comment by codex-zaydek** [comment]
Hey @katoquro, I made a reference Figma version [here](https://www.figma.com/file/vfjBXrSSOCgmVEX5fdvV4L/Heroicons-v0.1-(2abb814)). I cleaned it up as much as possible without altering the icons. Note that the outlined versions of `currency-dollar` and `ticket` are still broken (see #11), but I’ve labeled them with a warning symbol and a comment to the outstanding issue.

👉 https://www.figma.com/file/vfjBXrSSOCgmVEX5fdvV4L/Heroicons-v0.1-(2abb814)

<img width="1440" alt="Screen Shot 2020-03-03 at 8 39 24 PM" src="https://user-images.githubusercontent.com/58870766/75772363-232bc900-5d8f-11ea-92e4-00c58a6d967a.png">

Feel free to copy, clone it, etc. I hope this helps in the meantime!
